netrd/reconstruction/correlation_matrix.py

```diff
@@ -0,0 +1,58 @@
+"""
+correlation_matrix.py
+---------------------
+
+Reconstruct a network from time series data using a regularized form of the
```

Comment:
Please move this piece of documentation to line 17 as the first line of documentation of the `fit` method. Also add your name/user/email to the top of the file as author.